src/System.Windows.Forms/src/System/Windows/Forms/Layout/DefaultLayout.cs

```diff
@@ -270,38 +352,70 @@ private static Rectangle GetAnchorDestination(IArrangedElement element, Rectangl
                 }
             }
 
-            Debug.WriteLineIf(CompModSwitches.RichLayout.TraceInfo, "\t\t...new anchor dim (l,t,r,b) {"
-                                                                      + (left)
-                                                                      + ", " + (top)
-                                                                      + ", " + (right)
-                                                                      + ", " + (bottom)
-                                                                      + "}");
+            Debug.WriteLineIf(CompModSwitches.RichLayout.TraceInfo, $"\t\t...new anchor dim (l,t,r,b) {{{left}, {top}, {right}, {bottom}}}");
 
             return new Rectangle(left, top, right - left, bottom - top);
         }
 
-        private static void LayoutAnchoredControls(IArrangedElement container)
+        /// <summary>
+        ///  Determines if AnchorLayoutV2 should be used to compute anchors of the element
+        ///  and layout child anchored elements with V2 version.
```

Comment:
```suggestion
        ///  and to layout child anchored elements with V2 version.
```

What's a "child anchored elements"? Does it mean "anchored children controls"?